src/test/java/teammates/test/cases/webapi/SearchStudentsActionTest.java

```diff
@@ -137,7 +137,6 @@ public void execute_instructorSearchGoogleId_matchOnlyStudentsInCourse() {
     }
 
     @Override
-    @Test
```

Comment:
Why is this removed?

Reply:
I think it was done by my IDE which I didn't notice/verify. Will revert, thanks for spotting that